packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -17,12 +31,38 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
   const { register, handleSubmit, trigger, formState } = useForm<TotpForm>({
     mode: 'onTouched',
   });
-
   const isValidTotpFormat = (totp: string) => /\d{6}/.test(totp);
+
+  const alertBar = useAlertBar();
+  const [qrCodeUrl, setQrCodeUrl] = useState<string>();
+  const [secret, setSecret] = useState<string>();
+  const [recoveryCodes, setRecoveryCodes] = useState<string[]>([]);
+
   const onSubmit = ({ totp }: TotpForm) => {};
 
+  const [createTotp] = useMutation(CREATE_TOTP_MUTATION, {
+    onCompleted: (x) => {
+      setQrCodeUrl(x.createTotp.qrCodeUrl);
+      setSecret(x.createTotp.secret);
+      setRecoveryCodes(x.createTotp.recoveryCodes);
+    },
+    onError: () => {
+      alertBar.error('There was a problem retrieving your code.');
+    },
+  });
+
+  useEffect(() => {
+    createTotp({ variables: { input: {} } });
```

Comment:
with an unverified session I get this screen...
<img width="719" alt="Screen Shot 2020-10-19 at 12 37 32 PM" src="https://user-images.githubusercontent.com/87619/96503446-1ec1fc80-1208-11eb-9182-ab71e19c76ef.png">

I think we need to put this call behind the guard somehow.